src/extractor.rs

```diff
@@ -764,6 +764,9 @@ impl Default for EitherCollisionStrategy {
     }
 }
 
+///Determines Either extractor configuration
+///
+///By default uses default `EitherCollisionStrategy`
```

Comment:
```suggestion
///By default uses default `EitherCollisionStrategy::FastestSuccessful `
```

Reply:
Oh damn, I forgot to add variant itself, thx for noticing it